src/test/java/com/imgix/test/TestSrcSet.java

```diff
@@ -99,13 +97,13 @@ public void testNoParametersReturnsExpectedNumberOfPairs() {
     @Test
     public void testNoParametersDoesNotExceedBounds() {
         String minWidth = srcsetSplit[0].split(" ")[1];
-        String maxWidth = srcsetSplit[srcsetSplit.length-1].split(" ")[1];
+        String maxWidth = srcsetSplit[srcsetSplit.length - 1].split(" ")[1];
 
-        int minWidthInt = Integer.parseInt(minWidth.substring(0,minWidth.length()-1));
-        int maxWidthInt = Integer.parseInt(maxWidth.substring(0,maxWidth.length()-1));
+        int minWidthInt = Integer.parseInt(minWidth.substring(0, minWidth.length() - 1));
+        int maxWidthInt = Integer.parseInt(maxWidth.substring(0, maxWidth.length() - 1));
 
-        assert(minWidthInt >= 100);
-        assert(maxWidthInt <= 8192);
+        assertTrue(minWidthInt >= 100);
```

Comment:
Sonar-lint was not happy we were not explicitly using `assertTrue` here. This change was necessary in several places.